menu.h

```diff
@@ -58,17 +62,27 @@ STATUS_t initSection() {
     } else {
       return FAILURE;
     }
+
+  footerLine = initLine( "Timer: ", "Timer", NULL);
+  footerLine->val->setChar(footerLine->val, "");
```

Comment:
Double check this functionality. You are parsing a pointer to a char array to the pointer of char